ecs-cli/modules/cli/compose/context/ecs_context.go

```diff
@@ -19,22 +19,24 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/sirupsen/logrus"
 	ec2client "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/ec2"
 	ecsclient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/ecs"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/compose"
 	"github.com/docker/libcompose/project"
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
-// Context is a wrapper around libcompose.project.Context
-type Context struct {
+// ECSContext is a wrapper around libcompose.project.Context
+type ECSContext struct {
```

Comment:
Wasn't this already renamed to ECSContext in a previous PR? May be showing in diff do to rebasing/etc. ...